mesh/settings.py

```diff
@@ -140,9 +140,3 @@
 
 # Default primary key field type
 # https://docs.djangoproject.com/en/4.2/ref/settings/#default-auto-field
-
-DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
-
-STATICFILES_DIRS = [
```

Comment:
Why is this being deleted

Reply:
Sorry about that, was debugging and accidentally deleted these lines.